filebeat/module/elasticsearch/slowlog/ingest/pipeline-json.json

```diff
@@ -87,6 +87,27 @@
                 "ignore_missing": true
             }
         },
+        {
+            "rename": {
```

Comment:
Yes, it should be okay because of the `ignore_missing` flag set on this processor (see a couple lines below). In 9 we can completely remove this processor from the pipeline as, by then we won't expect to see any log lines with `doc_type` in them.